### DISCUSSION TITLE
Is there a way to remove the spacing between inlined elements when calling the AlignJustify() method?

### DISCUSSION THREAD

**Discussion by diegomarquez-real** [QuestPDF/QuestPDF#508] [General]
In my page content I would like to create a wrapping paragraph with a combination of normal text and weblink elements. AlignJustify works, except for the spacing in between elements. Is there a way to update the spacing or remove as a whole?

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. We apologize for the delayed response. Thank you for taking the time to reach out.

Since this discussion was opened, the text rendering engine has been significantly overhauled. You can now mix regular text and hyperlinks within a single `.Text()` block using `.Hyperlink()` spans, combined with `.Justify()` for justified alignment — this handles spacing naturally without needing to use `Inlined` elements.

You can find the documentation here:
https://www.questpdf.com/api-reference/text/paragraph-style.html
https://www.questpdf.com/api-reference/hyperlink.html

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
